concent_api/concent_api/urls.py

```diff
@@ -13,12 +13,28 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
+from django.conf      import settings
 from django.conf.urls import url, include
 from django.contrib   import admin
 
 import core.urls
+import gatekeeper.urls
 
-urlpatterns = [
-    url(r'^api/v1/', include(core.urls, namespace = 'core')),
-    url(r'^admin/',  admin.site.urls),
-]
+urlpatterns = []
+
+assert hasattr(settings, 'CONCENT_FEATURES') is True, "CONCENT_FEATURES variable must be defined in django settings."
```

Comment:
`is True` is redundant here. `hasattr(settings, 'CONCENT_FEATURES')` returns a boolean value so it's sufficient.